string action_label # The label used to be displayed in the action menu item
google_chat_ros/OnClick on_click
